travis.sh

```diff
@@ -4,13 +4,27 @@ set -exo pipefail
 
 N=2
 
-make -j$N -C src -f posix.mak HOST_DMD=$DMD
-make -j$N -C src -f posix.mak dmd.conf HOST_DMD=$DMD
 git clone --depth=1 https://github.com/D-Programming-Language/druntime.git ../druntime
 git clone --depth=1 https://github.com/D-Programming-Language/phobos.git ../phobos
+
+make -j$N -C src -f posix.mak HOST_DMD=$DMD all
+make -j$N -C src -f posix.mak HOST_DMD=$DMD dmd.conf
 make -j$N -C ../druntime -f posix.mak
 make -j$N -C ../phobos -f posix.mak
 
+if [ $SELF_COMPILE -eq 1 ]; then
```

Comment:
Wouldn't it be better to self-compile once then diff the binaries?  Compiling again with identical binaries seems like a waste of time...